src/component/ContactTable/ContactTable.module.scss

```diff
@@ -0,0 +1,32 @@
+.add_button_round {
```

Comment:
Hi, I understand that you use material-UI component,
But is it ok to use  BEM naming method for your CSS?

Like: .addButton--round, .addButton--box?

Reply:
Hi Karlok, this is an interesting topic. I think the naming format is not that formal as well, but I haven't figure out what is the optimal naming convention in our project yet. So I just use it temporarily before we made a decision.

The reason I don't use BEM is that we are using CSS module for adding styles like: 
`<button className={styles.add_button_round}>`
If we use "--" in the class name as BEM required, it will cause problems when accessed within JavaScript. 

In addition, the class names displayed in the "inspect" of Chrome browser are hashed under current configuration, and the scope of class names is limited in the components under the CSS module. Thus, it seems BEM is not the best convention in this situation.

I think we can have more research on this topic and discuss in the meetings.